### ISSUE TITLE
Add good problems you find on Hackerrank

### ISSUE THREAD

**Issue by dheeraj-2000** [dheeraj-2000/30-Days-Of-Code#19]
### Keep such code in folder name as **Other codes of Hackerrank**

**Comment by ke3peR007** [comment]
I would like to work on it please

**Comment by ke3peR007** [comment]
I would like to work on it

**Comment by Shobhit-aryan** [comment]
I would like to contribute to it. Can you assign it to me?

**Comment by dheeraj-2000** [comment]
@ke3peR007 @Shobhit-aryan go ahead, I've assigned to both of you.

**Comment by techakhilc47** [comment]
I would like to contribute to it. Can you assign it to me?

**Comment by RidhimaKohli** [comment]
Can I add a problem solution in python ?

**Comment by dheeraj-2000** [comment]
@RidhimaKohli yes you can, also please put the problem link within your code. And if, it's doesn't belongs to Hackerrank, then contribute to [this repo](https://github.com/dheeraj-2000/dsalgo)

**Comment by durgeshahire07** [comment]
i would like to contribute to it. can you assign me this issue?

**Comment by dheeraj-2000** [comment]
@durgeshahire07 Sure, follow the guidlines and send a PR

**Comment by durgeshahire07** [comment]
please check and review #28

**Comment by achalesh27022003** [comment]
Well I have added one (Organising Containers of Balls). I can contribute more problems' solution with good logic and problem solving.,[HackerRank Profile](https://www.hackerrank.com/Achal_2702). Can I add more? Pls merge previous one too. Thanks.

**Comment by omkarvw** [comment]
I would like to contribute to this. So could you please assign it to me ?

**Comment by MinalGhate** [comment]
Hello sir I would like to contribute on this issue

**Comment by kanikarajpal** [comment]
Can you please assign this to me?

**Comment by akash-1107** [comment]
Please Assign this to me
